server/src/main/java/com/broadcom/lsp/cobol/core/model/RangeModel.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package com.broadcom.lsp.cobol.core.model;
+
+import lombok.Value;
+import org.eclipse.lsp4j.Range;
+
+/**
+ * This model represents a range of a statement (i.e. section or paragraph)
+ */
+@Value
+public class RangeModel {
```

Comment:
The class content confuses me. Looks like it is a range with start and end. But it consists of ranges.
What do you expect as a result of `rangeModel.getStart().getEnd()` or `rangeModel.getEnd().getStart()`?